src/components/Avatar/Avatar.tsx

```diff
@@ -65,6 +76,9 @@ export const Avatar: FC<AvatarProps> = React.forwardRef(
             fontSize = '18px',
             iconProps,
             children,
+            hashingFunction,
+            theme,
```

Comment:
can we make default blue

Reply:
If we keep a default theme, we'll never be able to use the hashing function on the basis of priority order we had decided